content/efcore/getting-started/samples/querying/loading-all-entities/Program.cs

```diff
@@ -9,7 +9,8 @@ public static void Main()
         using (var context = new LibraryContext())
         {
             var books = context.Books.ToList();
-            foreach (Book book in books)
+            Console.WriteLine("Books in library -");
```

Comment:
Colons are used to introduce lists, not dashes:

- http://www.wheaton.edu/Academics/Services/Writing-Center/Writing-Resources/Semicolons-Colons-and-Dashes
- http://writingcenter.unc.edu/handouts/semi-colons-colons-and-dashes/
- https://www.pugetsound.edu/academics/academic-resources/cwlt/writing-advisor-schedule/writing-resources/colon-and-dashes/
- https://www.bethel.edu/undergrad/academics/support/colon-dash.pdf